pkg/storaged/crypto-keyslots.jsx

```diff
@@ -622,15 +622,13 @@ export const TangKeyVerification = ({ url, adv }) => {
             <Text component={TextVariants.p}>{_("Check the key hash with the Tang server.")}</Text>
 
             <Text component={TextVariants.h3}>{_("How to check")}</Text>
-            <Text component={TextVariants.p}>
-                {_("In a terminal, run: ")}
-                <ClipboardCopy hoverTip={_("Copy to clipboard")}
-                               clickTip={_("Successfully copied to clipboard!")}
-                               variant="inline-compact"
```

Comment:
Hmm, shouldn't `inline-compact` turn the ClipboardCopy into a span?

Reply:
Ah, no, it changes it to `display: inline`, but keeps it a div.

Reply:
Yes, but the more important point is that the `<p>` was a block element, and by default the next element gets rendered below it. `<span>` renders the next element to the right.